_includes/chapter4.html

```diff
@@ -144,7 +144,7 @@ <h2>Rolling forward with hotfixes</h2>
 </p>
 <div><img alt="Hotfix" src="images/octopus/hotfixdeployment.png"/></div>
 <p>
-    A hotfix channel gives us the ability to push a release to any environment. If we still need some verification of the release in a non-production environment, we can deploy to <b>Development</b> or <b>Test</b>. But if we really need to promote a deployment straight to <b>Production</b>, this is now an option.
+    A hotfix channel gives us the ability to push a release to any environment. If we still need verification of the release in a non-production environment, we can deploy to <b>Development</b> or <b>Test</b>. But if we really need to promote a deployment straight to <b>Production</b>, this is now an option.
```

Comment:
@wordlee where it says "But if we really need to promote a deployment straight to <b>Production</b>, this is now an option." - Should production be with a lower case p?

Reply:
It depends on whether the name is a common or proper noun. I'll take a look at the convention Matt has used throughout the book and correct as needed.

Reply:
I think in this instance the environments have been called Test and Production.